computation/src/test/java/com/powsybl/computation/DefaultComputationManagerConfigTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.computation;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import com.powsybl.commons.config.InMemoryPlatformConfig;
+import com.powsybl.commons.config.MapModuleConfig;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.nio.file.FileSystem;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class DefaultComputationManagerConfigTest {
+
+    public static class FirstComputationManagerFactory implements ComputationManagerFactory {
+        @Override
+        public ComputationManager create() {
+            return Mockito.mock(ComputationManager.class);
+        }
+    }
+
+    public static class SecondComputationManagerFactory implements ComputationManagerFactory {
+        @Override
+        public ComputationManager create() {
+            return Mockito.mock(ComputationManager.class);
+        }
+    }
+
+    private FileSystem fileSystem;
+
+    @Before
+    public void setUp() {
+        fileSystem = Jimfs.newFileSystem(Configuration.unix());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        fileSystem.close();
+    }
+
+    @Test
+    public void test() {
+        DefaultComputationManagerConfig config = new DefaultComputationManagerConfig(FirstComputationManagerFactory.class,
+                                                                                     SecondComputationManagerFactory.class);
+        assertEquals("DefaultComputationManagerConfig(shortTimeExecutionComputationManagerFactoryClass=com.powsybl.computation.DefaultComputationManagerConfigTest$FirstComputationManagerFactory, longTimeExecutionComputationManagerFactoryClass=com.powsybl.computation.DefaultComputationManagerConfigTest$SecondComputationManagerFactory)", config.toString());
```

Comment:
You can probably generate this class name instead of using an hard-coded string value (x4)

Reply:
Done